src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -533,4 +533,97 @@ lemma norm_noninc_of_isometry (hf : isometry f) : f.norm_noninc :=
 
 end isometry
 
+variables {V₄ W₁ W₂ W₃ : Type*} [semi_normed_group V₄] [semi_normed_group W₁] [semi_normed_group W₂]
+  [semi_normed_group W₃]
+variables (f) (g : normed_group_hom V W)
+variables {f₁ g₁ : normed_group_hom V₁ W₁}
+variables {f₂ g₂ : normed_group_hom V₂ W₂}
+variables {f₃ g₃ : normed_group_hom V₃ W₃}
+
+/-- The equalizer of two morphisms `f g : normed_group_hom V W`. -/
+def equalizer := (f - g).ker
```

Comment:
Is this worth mentioning in the module docstring?

Reply:
Done it.